Analyzer/plugins/Analyzer.cc

```diff
@@ -1818,6 +1830,13 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
                                 mustBeInside, MaxStripNOM, correctFEDSat, crossTalkInvAlgo = 1, dropLowerDeDxValue = 0.0, 0, useTemplateLayer_);
                     
                     dedxIas_FullTracker = dedxIas_FullTrackerTmp.numberOfMeasurements() > 0 ? &dedxIas_FullTrackerTmp : nullptr;
+
+                    //globalIas_ no BPIXL1
+                    dedxIas_noL1Tmp =
+                    computedEdx(track->eta(),run_number, year, dedxHits, dEdxSF, localdEdxTemplates = dEdxTemplatesPU[i], usePixel = true, useStrip = true, useClusterCleaning, useTruncated = false,
+                           mustBeInside, MaxStripNOM, correctFEDSat, crossTalkInvAlgo = 1, dropLowerDeDxValue = 0.0, 0, useTemplateLayer_,skipPixelL1 = true, skip_templates_ias = 2);
+
+                    dedxIas_noL1 = dedxIas_noL1Tmp.numberOfMeasurements() > 0 ? &dedxIas_noL1Tmp : nullptr;
```

Comment:
I think this is a duplication of line 1785, no?

Reply:
I don't see what is on L1785. If it is related to dedxIas_FullTracker, then my understanding is that dedxIas_FullTracker is with BPIXL1 and dedxIas_noL1 is without.

Reply:
this is line 1785
```
dedxIas_noL1Tmp =
            computedEdx(track->eta(),run_number, year, dedxHits, dEdxSF, localdEdxTemplates = dEdxTemplates, usePixel = true, useStrip = true, useClusterCleaning, useTruncated = false,
                   mustBeInside, MaxStripNOM, correctFEDSat, crossTalkInvAlgo = 1, dropLowerDeDxValue = 0.0, 0, useTemplateLayer_,skipPixelL1 = true, skip_templates_ias = 2);
```

Reply:
no there is no duplication. It is related to the change from Raphael, we are in the if condition `        if(!puTreatment_ || CreateAndRunGitemplates_){  ` for PU treatment of the Ias templates. on line 1785 you see ` localdEdxTemplates = dEdxTemplates `  while on line 1836 you see ` localdEdxTemplates = dEdxTemplatesPU[i]  `. I add my new variable in the same way the other variables are described with the new functionality from Raphael PR.